### PR TITLE
Update dev

### PR DESCRIPTION
**Pull request by SuperPenguin**
* Fix GraphyManager.cs enable disable bug when not initialized yet
* Fix guistyle caching

### CODE REVIEW

File: Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Dummy/SODummy.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using UnityEngine;
+
+namespace Tayx.Graphy
+{
+    public class SODummy : ScriptableObject
```
Comment (Tayx94) [inline Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Dummy/SODummy.cs]: Can you rename the class to have G_ as a prefix? That way it's consistent with the rest of the code :)

File: Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Editor/GraphyEditorStyle.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    {
+        public static Texture2D m_logoTexture = null;
+        public static GUISkin m_skin = null;
+        public static GUIStyle m_headerStyle1 = null;
+        public static GUIStyle m_headerStyle2 = null;
+        public static GUIStyle foldoutStyle = null;
+
+        private static string path;
+
+        static GraphyEditorStyle()
```
Comment (Tayx94) [inline Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Editor/GraphyEditorStyle.cs]: can you add "public"?

File: Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Editor/GraphyEditorStyle.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Tayx.Graphy
+{
+    internal static class GraphyEditorStyle
```
Comment (Tayx94) [inline Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Editor/GraphyEditorStyle.cs]: Can you add #regions ?

File: Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Editor/GraphyEditorStyle.cs (hunk 2 above)
Comment (SuperPenguin) [reply Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Editor/GraphyEditorStyle.cs]: static constructor can't have access modifier

Comment (Tayx94) [reply Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Editor/GraphyEditorStyle.cs]: My bad, did not notice it was the constructor!

**Pull request merged**